storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -96,37 +91,57 @@ include::modules/lvms-monitoring-logical-volume-manager-operator.adoc[leveloffse
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-integrating-software-raid-arrays_logical-volume-manager-storage[Integrating software RAID arrays with {lvms}]
 
-include::modules/lvms-scaling-storage-of-single-node-openshift-cluster-using-rhacm.adoc[leveloffset=+2]
+include::modules/lvms-scaling-storage-of-clusters-using-web-console.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html/install/installing#installing-while-connected-online[Red Hat Advanced Cluster Management for Kubernetes: Installing while connected online]
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-unsupported-devices_logical-volume-manager-storage[Devices not supported by {lvms}]
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-integrating-software-raid-arrays_logical-volume-manager-storage[Integrating software RAID arrays with {lvms}]
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]
+
+include::modules/lvms-scaling-storage-of-clusters-using-rhacm.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-unsupported-devices_logical-volume-manager-storage[Devices not supported by {lvms}]
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-integrating-software-raid-arrays_logical-volume-manager-storage[Integrating software RAID arrays with {lvms}]
 
-* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the `LVMCluster` custom resource]
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]
 
-include::modules/lvms-scaling-storage-expand-pvc.adoc[leveloffset=+2]
+// Expanding PVCs
+include::modules/lvms-scaling-storage-expand-pvc.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-single-node-openshift-cluster_logical-volume-manager-storage[Scaling up storage by adding capacity to your cluster]
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-clusters-using-web-console_logical-volume-manager-storage[Scaling up storage by adding capacity to your cluster]
```

Comment:
Link text is different here:
```suggestion
* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-clusters-using-web-console_logical-volume-manager-storage[Scaling up the storage of clusters by using the web console]
```
(remember to remove "by" here if you accept my suggestion to remove it from the module heading)

Reply:
Removed these references and add the following reference:

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-about-scaling-storage-of-cluster_logical-volume-manager-storage[Scaling up the storage of clusters]

As this section includes all three ways of scaling up the storage.